src/api/index.js

```diff
@@ -1,10 +1,22 @@
 import axios from 'axios';
+// const CORS_PROXY = 'https://cors-anywhere.herokuapp.com/';
+const BASE_URL = `/api`;
 
-export async function getSomething() {
-  try {
-    const { data } = await axios.get('/api');
-    return data;
-  } catch (error) {
-    throw error;
-  }
-}
\ No newline at end of file
+export const callApi = async ({ path, method, token }, body = null) => {
+	const axiosConfig = {
+		url: `${BASE_URL + path}`,
+		method: `${method}`,
+		data: body
+	};
+
+	if (token) {
+		axiosConfig.headers = { Authorization: `Bearer ${token}` };
+	}
+
+	try {
+		const { data } = await axios(axiosConfig);
+		return data;
```

Comment:
In future, maybe error handle here. Like if the data doesn't come back, or if the data is in a specific format, throw an error (i.e. `setStatus` in React).